pageserver/src/layered_repository.rs

```diff
@@ -583,6 +585,13 @@ pub struct LayeredTimeline {
     // because current_logical_size is consistent on last_record_lsn, not ondisk_consistent_lsn
     // NOTE: current_logical_size also includes size of the ancestor
     current_logical_size: AtomicUsize, // bytes
+
+    // needed to exclude calls to .with_label_values and labels formatting to strings
+    // every time we need to change a metric
```

Comment:
Suggested rewording, :
```
    // To avoid calling .with_label_values and formatting the tenant and timeline IDs to strings
    // every time the logical size is updated, keep a direct reference to the Gauge here.
```